src/ahbicht/models/mapping_results.py

```diff
@@ -96,10 +96,11 @@ def check_max_greater_or_equal_than_min(instance: "Repeatability", attribute, va
     """
     assert that 0<=min<max and not both min and max are 0
     """
-    if not 0 <= instance.min_occurrences <= instance.max_occurrences:
-        raise ValueError(f"0≤n≤m is not fulfilled for n={instance.min_occurrences}, m={instance.max_occurrences}")
-    if instance.min_occurrences == instance.max_occurrences == 0:
-        raise ValueError("not both min and max occurrences must be 0")
+    if instance.max_occurrences != "n":
+        if not 0 <= instance.min_occurrences <= instance.max_occurrences:
+            raise ValueError(f"0≤n≤m is not fulfilled for n={instance.min_occurrences}, m={instance.max_occurrences}")
+        if instance.min_occurrences == instance.max_occurrences == 0:
+            raise ValueError("not both min and max occurrences must be 0")
```

Comment:
Ich würde ungern alle Checks deaktivieren, wenn `n` angegeben ist.
```suggestion
    max_occurrences = math.inf if instance.max_occurrences == "n" else instance.max_occurrences
    if not 0 <= instance.min_occurrences <= max_occurrences:
        raise ValueError(f"0≤n≤m is not fulfilled for n={instance.min_occurrences}, m={max_occurrences}")
    if instance.min_occurrences == max_occurrences == 0:
        raise ValueError("not both min and max occurrences must be 0")
```
Für den Vorschlag hier müsste man aber noch das `math` module oben importieren.